docs/how-to-guides/manage-connections.md

```diff
@@ -210,6 +210,56 @@ On the VS Code primary sidebar > prompt flow pane. You can find the connections
 :::
 ::::
 
+## Consume connections on cloud (Azure AI)
```

Comment:
I think this topic should be posed on cloud side. we can add a link to there

Reply:
have moved to cloud section